packages/react-native-bridge/README.md

```diff
@@ -0,0 +1,44 @@
+
+# react-native-gutenberg-bridge
+
+## Getting started
+
+`$ npm install react-native-gutenberg-bridge --save`
```

Comment:
I don't think it published to npm yet, nor will be in the near future, should it be clarified?

Reply:
Updated the README suggesting to use it locally for now install of npm install.